### PR TITLE
Hunter 3.05.01

### PR DESCRIPTION
**Pull request by xsacha**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -10,6 +10,14 @@
 
 cmake_minimum_required(VERSION 2.8.11)
```
Comment (ruslo) [inline CMakeLists.txt]: For Hunter minimum is 3.0:
* https://docs.hunter.sh/en/latest/quick-start/cmake.html

**Review by ruslo — CHANGES_REQUESTED** [review]

File: CMakeLists.txt [hunk 2]
```diff
@@ -68,21 +72,18 @@ else()
     add_subdirectory(.cppan)
 endif()
 
+hunter_add_package(OpenCL)
 find_package(OpenCL QUIET)
```
Comment (ruslo) [inline CMakeLists.txt]: `find_package(OpenCL REQUIRED CONFIG)`:
* https://github.com/ruslo/hunter/wiki/pkg.opencl

**Review by ruslo — CHANGES_REQUESTED** [review]

**Comment by xsacha** [comment]
I think that is everything?

**Review by ruslo — APPROVED** [review]

**Comment by ruslo** [comment]
https://github.com/hunter-packages/tesseract/releases/tag/v3.05.01-hunter-0

**Pull request merged**